devel/ansible/roles/datanommer/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Install RPM packages
+  dnf:
+      name:
+      - fedora-messaging
+      - gcc
+      - git
+      - poetry
+      - python3-devel
+      - python3-pip
+      - vim
+      - krb5-devel
+      state: present
+
+- name: Create a directory for the virtualenv
+  file:
+    name: /srv/venv
+    state: directory
+    mode: 0755
+    owner: vagrant
+    group: vagrant
+
+- name: Create the virtualenv and install poetry
+  pip:
+    name: poetry
+    virtualenv: /srv/venv
+  become: yes
+  become_user: vagrant
+
+- name: Tell poetry not to create a virtualenv
```

Comment:
I'm not sure that's necessary, poetry should not create an additional virtualenv when it's running from a virtualenv already

Reply:
Which means that we should source `/srv/venv/bin/activate` from the vagrant user's `.bashrc`.

Reply:
Fair enough -- was just looking at fasjson's vagrant setup, and in there we alias /srv/venv/bin/potery to poetry: https://github.com/fedora-infra/fasjson/blob/dev/ansible/roles/fasjson/files/.bashrc

does that achieve the same thing? or would  you rather activating the venv with source in the bashrc?

Reply:
I think it's pretty much the same. But it feels like enabling the virtualenv is closer to the "nominal path" than making an alias and setting the poetry configuration. Maybe it's just me :-) Both would probably work.